http-clients/src/test/java/org/triplea/http/client/moderator/toolbox/ModeratorToolboxFeignClientTest.java

```diff
@@ -0,0 +1,202 @@
+package org.triplea.http.client.moderator.toolbox;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.Is.is;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import javax.annotation.Nonnull;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.triplea.http.client.HttpClient;
+import org.triplea.http.client.HttpClientTesting;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import lombok.Builder;
+import ru.lanwen.wiremock.ext.WiremockResolver;
+import ru.lanwen.wiremock.ext.WiremockUriResolver;
+
+@ExtendWith({
+    WiremockResolver.class,
+    WiremockUriResolver.class
+})
+class ModeratorToolboxFeignClientTest {
+  private static final String SUCCESS_JSON = "SUCCESS";
+  private static final String INPUT_JSON = "word";
+
+
+  private static final String EPOCH_SECOND = "1559845434.585923000";
+  private static final ModeratorEvent EVENT_1 = ModeratorEvent.builder()
+      .date(Instant.ofEpochSecond(1559845434, 585923000))
+      .actionTarget("Malaria is a cloudy pin.")
+      .moderatorAction("Jolly roger, real pin. go to puerto rico.")
+      .moderatorName("All parrots loot rainy, stormy fish.")
+      .build();
+
+  private static final String MDOERATOR_EVENT_JSON_RESPONSE =
+      "[{\"date\":" + EPOCH_SECOND + ","
+          + "\"moderatorName\":\"" + EVENT_1.getModeratorName() + "\","
+          + "\"moderatorAction\":\"" + EVENT_1.getModeratorAction() + "\","
+          + "\"actionTarget\":\"" + EVENT_1.getActionTarget() + "\"}]";
+
+  private static final Map<String, Object> headerMap;
+
+  private static final String HEADER_KEY = "key";
+  private static final String HEADER_VALUE = "value";
+
+  static {
+    headerMap = new HashMap<>();
+    headerMap.put(HEADER_KEY, HEADER_VALUE);
+  }
```

Comment:
Same here with `Collections#singletonMap`

Reply:
Ditto response, API key password is coming along soon.